ansible/tests/unit/v2/values-secret-v2-base.yaml

```diff
@@ -0,0 +1,42 @@
+# Needed to specify the new format (missing version means old version: 1.0 by default)
+version: 2.0
+# NEVER COMMIT THESE VALUES TO GIT
+
+# global + backend is the default. No need to specify it but one day we could have an override implementing it for, say, cyberark
+global:
+  backend: vault
+
+vaultPolicies:
+  # https://developer.hashicorp.com/vault/docs/concepts/password-policies
+  basicPolicy: |
+    length=10
+    rule "charset" { charset = "abcdefghijklmnopqrstuvwxyz" min-chars = 1 }
+    rule "charset" { charset = "ABCDEFGHIJKLMNOPQRSTUVWXYZ" min-chars = 1 }
+    rule "charset" { charset = "0123456789" min-chars = 1 }
+
+  advancedPolicy: |
+    length=20
+    rule "charset" { charset = "abcdefghijklmnopqrstuvwxyz" min-chars = 1 }
+    rule "charset" { charset = "ABCDEFGHIJKLMNOPQRSTUVWXYZ" min-chars = 1 }
+    rule "charset" { charset = "0123456789" min-chars = 1 }
+    rule "charset" { charset = "!@#$%^&*" min-chars = 1 }
+
+secrets:
+  - name: config-demo
+    vaultPrefixes:
+    - region-one
+    - snowflake.blueprints.rhecoeng.com
+    fields:
+    - name: secret
+      onMissingValue: generate # One of: error,generate,prompt
+      vaultPolicy: basicPolicy
+    - name: secret2
+      value: null
+      onMissingValue: prompt
+    - name: ca_crt
+      path: /tmp/ca.crt
+      onMissingValue: error # One of error, prompt (for path). generate makes no sense for file
```

Comment:
Lets leave out the internal "note to self" comments :)

Reply:
Aye, in the unit test files that is a fair point. Will do